docs/workloads/batch/configuration.md

```diff
@@ -23,6 +23,10 @@ predictor:
   env: <string: string>  # dictionary of environment variables
   log_level: <string>  # log level that can be "debug", "info", "warning" or "error" (default: "info")
   shm_size: <string>  # size of shared memory (/dev/shm) for sharing data between multiple processes, e.g. 64Mi or 1Gi (default: Null)
+  dependency_path: # (optional)
+    pip: <string>  # path to requirements.txt (default: requirements.txt)
+    conda: <string>  # path to conda-packages.txt (default: conda-packages.txt)
+    shell: <string>  # path to a shell script for system package installation (default: dependencies.sh)
```

Comment:
I think we should also consider adding `.cortexignore` to the list of things that can be configured, since otherwise, a change in a different API's `requirements.txt` would cause an API to be considered updated. What do you think?

Reply:
i am not following this